### PR TITLE
72991: Added screenshot. Fixed passive language. Moved note. All for asigra.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/services.rst [hunk 1]
```diff
@@ -254,6 +249,18 @@ Asigra management application.
    the application to open *DSOP.jnlp* with, select Java Web Start
    Launcher (:command:`javaws`).
 
+While the service is running, the :guilabel:`Open DS-Operator Web Interface`
+button appears in :menuselection:`Services --> Asigra DS-System`. Click
```
Comment (Mrt134) [inline userguide/services.rst]:
```suggestion
button appears in
:menuselection:`Services --> Asigra DS-System`.
Click :guilabel:`Open DS-Operator Web Interface` to download and launch the
```

File: userguide/services.rst [hunk 2]
```diff
@@ -254,6 +249,18 @@ Asigra management application.
...
+:guilabel:`Open DS-Operator Web Interface` to download and launch the
```
Comment (Mrt134) [inline userguide/services.rst]:
```suggestion

```

File: userguide/services.rst (hunk 1 above)
Comment (Mrt134) [reply userguide/services.rst]: We're supposed to put :menuselection:`` items on their own line now (confirm with Warren)

File: userguide/services.rst [hunk 3]
```diff
@@ -238,13 +238,8 @@ Licensed Asigra Backup software can use %brand% as the storage backend.
...
 :menuselection:`Services --> Asigra DS-System`. When prompted to choose
 the :guilabel:`Base Filesystem`, select the dataset to store the Asigra
-backups then click :guilabel:`OK`. The system will automatically create
-the required database entries and start the service.
-
-While the service is running, the :guilabel:`Open DS-Operator Web Interface`
-button appears in :menuselection:`Services --> Asigra DS-System`. Click
-:guilabel:`Open DS-Operator Web Interface` to download and launch the
-Asigra management application.
+backups then click :guilabel:`OK`. The system automatically creates
```
Comment (wblock) [inline userguide/services.rst]: Take a breath here, and "the system" is ambiguous.  Fortunately, it is not needed:
```suggestion
backups, then click :guilabel:`OK`. Any required database entries are created and the service is started.
```

File: userguide/services.rst [hunk 4]
```diff
@@ -238,13 +238,8 @@ Licensed Asigra Backup software can use %brand% as the storage backend.
...
+the required database entries and starts the service.
```
Comment (wblock) [inline userguide/services.rst]:
```suggestion

```

File: userguide/services.rst [hunk 5]
```diff
@@ -254,6 +249,19 @@ Asigra management application.
...
+Asigra management application.
+
+.. _asigra settings:
+
+.. figure:: images/services-asigra.png
+   :width: 50%
+
+   Asigra settings
+
 Contact Asigra for further documentation on using DS-Operator.
```
Comment (wblock) [inline userguide/services.rst]: Can we add a link here?

**Pull request merged**